packages/tasit-identity-contract/src/GnosisSafe.test.js

```diff
@@ -63,11 +67,18 @@ describe("GnosisSafe", () => {
 
     anaWallet = createFromPrivateKey(anaPrivateKey);
 
+    // Not using Account.create() because tasit-action isn't a dependency of that package
```

Comment:
Should we make it a dep?

Reply:
Extracted to: https://github.com/tasitlabs/TasitSDK/issues/237

Reply:
Thanks. Resolving